### DISCUSSION TITLE
Why I can't find notes by title

### DISCUSSION THREAD

**Discussion by Marcosffn** [scambier/obsidian-omnisearch#112] [Q&A] [answered]
Many times I try to find a note by its title but I don't get the result I expected.

![Captura de tela de 2022-10-15 19-16-54](https://user-images.githubusercontent.com/67158009/196009670-615099fa-91cb-489c-a18d-c17bb8e9b72e.png)

I activated diacricts but they don't appear, I also tried to make the file name and declared alias the maximum weight but I still fail to get the notes I want or if I search the exact match of the title of the note they are not shown first.

**Comment by tekwizz123** [comment]
@Marcosffn Have you tried wrapping what you are searching for in double quotes? Otherwise without double quotes Obsidian will treat it as though you are searching for two separate items at the same time instead of one item with a space in its name.

**Comment by scambier** [comment, marked as answer]
- **Enable** the "Ignore Diacritics" setting
- **Disable** the "Persist cache on disk" setting
- Restart Obsidian

There are possibly an issue with diacritics, I have to check and will probably remove the setting altogether. Don't hesitate to report back if the results are still inconsistent, or if you have other examples.

**Reply by Marcosffn** [reply]
> * **Enable** the "Ignore Diacritics" setting
> 
>     * **Disable** the "Persist cache on disk" setting
> 
>     * Restart Obsidian
> 
> 
> There are possibly an issue with diacritics, I have to check and will probably remove the setting altogether. Don't hesitate to report back if the results are still inconsistent, or if you have other examples.

So I enabled "ignore diacritics" and now I can find the note in the print, I think its working properly now, thank you.
